speedtest/datadog_checks/speedtest/speedtest.py

```diff
@@ -0,0 +1,167 @@
+        }
+        """
+        # log for debugging, there is nothing sensitive here
+        self.log.debug("speedtest output: %s", payload)
+        print(payload)
+        if payload.get("type") != "result":
+            raise Exception("unexpected result type found: {}".format(payload.get("type")))
+
+        result_data = payload.get("result")
+        server_data = payload.get("server")
+        interface_data = payload.get("interface")
+        tags = (tags or []) + [
+            "isp:{}".format(payload.get("isp")),  # this will be normalized per tag standards
+            "interface_name:{}".format(interface_data.get("name")),
+            "server_id:{}".format(server_data.get("id")),
+            "server_name:{}".format(server_data.get("name")),
+            "server_country:{}".format(server_data.get("country")),
+            "server_host:{}".format(server_data.get("host")),
+        ]
+
+        event_msg = (
+            "[Test Results {test_id}]({test_url})\n\nServer: {server_name}({server_id}) - "
+            + "{server_location}, {server_country} at {server_host}:{server_port} ({server_ip}:{server_port})\n"
+            + "ISP: {isp}"
+        )
+
+        self.event(
+            {
+                "msg_title": "Speedtest Run {}".format(result_data.get("id")),
+                "msg_text": event_msg.format(
+                    test_id=result_data.get("id"),
+                    test_url=result_data.get("url"),
+                    server_name=server_data.get("name"),
+                    server_id=server_data.get("id"),
+                    server_location=server_data.get("location"),
+                    server_country=server_data.get("country"),
+                    server_host=server_data.get("host"),
+                    server_port=server_data.get("port"),
+                    server_ip=server_data.get("ip"),
+                    isp=payload.get("isp"),
+                ),
+                "tags": tags,
+            }
+        )
+
+        # ping data
+        ping_data = payload.get("ping")
+        self.gauge("speedtest.ping.jitter", float(ping_data.get("jitter")), tags)
+        self.gauge("speedtest.ping.latency", float(ping_data.get("latency")), tags)
+
+        download_data = payload.get("download")
+        self.gauge("speedtest.download.bandwidth", float(download_data.get("bandwidth")) * MEGABYTE_TO_MEBIBYTE, tags)
+        self.gauge("speedtest.download.bytes", float(download_data.get("bytes")), tags)
+        self.gauge("speedtest.download.elapsed", float(download_data.get("elapsed")), tags)
+
+        upload_data = payload.get("upload")
+        self.gauge("speedtest.upload.bandwidth", float(upload_data.get("bandwidth")) * MEGABYTE_TO_MEBIBYTE, tags)
+        self.gauge("speedtest.upload.bytes", float(upload_data.get("bytes")), tags)
+        self.gauge("speedtest.upload.elapsed", float(upload_data.get("elapsed")), tags)
+
+        self.gauge("speedtest.packet_loss", float(payload.get("packetLoss")), tags)
```

Comment:
```suggestion
        self.gauge("speedtest.packet_loss", float(payload.get("packetLoss", 0)), tags)
```

This line raises an exception when `packetLoss` isn't included in the output as with my tests on macOS and Ubuntu.